src/packages/migrate/src/utils/ensureDatabaseExists.ts

```diff
@@ -115,10 +115,36 @@ export async function askToCreateDb(
   }
 }
 
-function getDbLocation(credentials: DatabaseCredentials): string {
+export function getDbLocation(credentials: DatabaseCredentials): string {
   if (credentials.type === 'sqlite') {
     return credentials.uri!
   }
 
   return `${credentials.host}:${credentials.port}`
 }
+
+export function getDbinfoFromCredentials(
+  credentials,
+): {
+  dbName: string
+  dbType: 'MySQL' | 'PostgreSQL' | 'SQLite'
+  schemaWord: 'database'
+} {
+  const dbName = credentials.database
+  const dbType =
+    credentials.type === 'mysql'
+      ? 'MySQL'
+      : credentials.type === 'postgresql'
+      ? 'PostgreSQL'
+      : credentials.type === 'sqlite'
+      ? 'SQLite'
+      : credentials.type
+
+  const schemaWord = 'database'
+
+  return {
+    dbName,
+    dbType,
+    schemaWord,
+  }
+}
```

Comment:
This part would need a MSSQL equivalent

Reply:
Yes please! We actually need that :) Can you add it?

Reply:
Done in https://github.com/prisma/prisma/commit/c0c70e96ae2ba8dfe669b4cf9ceefec5ac9937be